docs/book/checkout.rst

```diff
@@ -0,0 +1,259 @@
+.. index::
+   single: Checkout
+
+Checkout
+========
+
+**Checkout** is a process that begins when the Customer decides to finish their shopping and transform their **Cart** into an **Order**.
```

Comment:
This phrasing is wrong. Customer doesn't want to transform anything, he just wants to buy stuffs. The things that actually happen in the back are not of his concern. Meaning, split this sentence into two. One as the customer's perspective and second with what happens in the kitchen. But that's just my opinion 😈